backend/src/middleware/notifications/notificationsMiddleware.spec.js

```diff
@@ -272,6 +275,9 @@ describe('notifications', () => {
         })
 
         describe('updates the post and mentions me again', () => {
+          const expectedUpdatedContent =
+            '<br>One more mention to<br><a data-mention-id="you" class="mention" href="/profile/you" target="_blank"><br>@al-capone<br></a><br>and again:<br><a data-mention-id="you" class="mention" href="/profile/you" target="_blank"><br>@al-capone<br></a><br>and again<br><a data-mention-id="you" class="mention" href="/profile/you" target="_blank"><br>@al-capone<br></a><br>'
+
```

Comment:
I have wired experiences with „variables“ directly in a describe block and not in an `before` or `it`. Just to let you know …